bolt-modules/boltlib/lib/puppet/functions/add_to_group.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'bolt/error'
+
+# Adds a target to specified inventory group.
+Puppet::Functions.create_function(:add_to_group) do
```

Comment:
This should also be added to the plan function documentation https://github.com/puppetlabs/bolt/blob/master/pre-docs/plan_functions.md

Reply:
I think that will be automatically generated?